### DISCUSSION TITLE
How would I add a Forgot Code Button

### DISCUSSION THREAD

**Discussion by ncaatrackstar** [welissonArley/Packages.MAUI#12] [Q&A]
Hello. Just letting you know that I love using your CodePage control. One thing that I cannot figure out is how to add a custom button to the bottom of the page so that when a user clicks it I can call some custom code. In this case a Forgot Code button that will allow a user to reset their Pincode. Any help would be appreciated.

**Comment by welissonArley** [comment]
Hi @ncaatrackstar thanks for your patience. I moved across the ocean and this took me some time. I'm going to restart and fix bugs and implement new features this week :)
